### DISCUSSION TITLE
Split right option does now work

### DISCUSSION THREAD

**Discussion by stefaniecg** [pulsar-edit/.github#127] [General]
Split right option does now work when one wants to move a tab to the right side. 
Work around, is move to the left, that does work, but right does not. 

Please fix, thank you

**Comment by Daeraxa** [comment]
If this is an issue (i.e. the application is not behaving correctly) then we need to know additional details asked by the issue template (https://github.com/pulsar-edit/pulsar/issues/new/choose).

However I can't see any issues with split right on my install so we need to know all the info to replicate it, OS, version etc.

**Comment by confused-Techie** [comment]
Since this is an issue created in the wrong location, without enough information to work from, saying there is an issue that we can't reproduce, and the op hasn't responded in over a week, how do we feel about deleting this discussion?

**Comment by stefaniecg** [comment]
@confused-Techie, hi thank you for replying to this post. I did investigate a bit, and it turns out the move right does not work but only for PDF files, with other files it does. So i'm not sure if the error is only in the package that enables pdf view in Pulsar. 
If you think is better to close this post, we can do so. :)

**Reply by confused-Techie** [reply]
We don't have to close it, we just didn't have a lot to go off of initially. 

Now I may be wrong, but what package are you using to view PDFs? I don't think that is one of our core packages.

Although with that said the Split right functionality is built into the core of the editor, so there may be a weird interaction between whatever package that is and Pulsars "Split Right" functionality, and if that is the case I would be interested to take a look to see what is at fault

**Reply by stefaniecg** [reply]
@confused-Techie , hi so the PDF package i'm using is the [pdfs-viewer 1.3.0](https://web.pulsar-edit.dev/packages/pdfjs-viewer). 
and there the `split right, left, up, down` does not work for me.

**Reply by confused-Techie** [reply]
So I'm able to test and confirm that when using `pdfs-viewer` that none of the split options work.

Since it's only occurring in this one tab for this package I'm inclined to believe the issue is on there end and there is only so much we can do.

If any maintainers of that package or anyone wants to open a PR for that package I am also seeing this logged in the error console when using this package 

```
offsetParent is not set -- cannot scroll
```
